tests/Behat/Content/ContentStorage.php

```diff
@@ -0,0 +1,97 @@
+<?php
```

Comment:
ContentStorage is useful because it can store not only sub entities but paragraph also.
But I'm not sure that tests/Behat/Content/ folder is suitable place for it.